x/staking/simulation/genesis_test.go

```diff
@@ -9,16 +9,18 @@ import (
 
 	"github.com/cosmos/cosmos-sdk/codec"
 	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
+	cryptocodec "github.com/cosmos/cosmos-sdk/crypto/codec"
 	"github.com/cosmos/cosmos-sdk/types/module"
 	simtypes "github.com/cosmos/cosmos-sdk/types/simulation"
 	"github.com/cosmos/cosmos-sdk/x/staking/simulation"
 	"github.com/cosmos/cosmos-sdk/x/staking/types"
 )
 
-// TestRandomizedGenState tests the normal scenario of applying RandomizedGenState.
+// Test tests the normal scenario of applying RandomizedGenState.
```

Comment:
I prefer the old comment

Reply:
I prefer the old comment too.

Reply:
I don't recall that change. Probably during some renames ;). Rolling back.